1 file with changes
===================

1) src/short_open_tag_with_import_name.php:0

    ---------- begin diff ----------
@@ -1,10 +1,12 @@
 <?php
+
+use PhpParser\Node\Stmt\Expression;
 function test_callback2($newNode) {
-  return new \PhpParser\Node\Stmt\Expression($newNode);
+  return new Expression($newNode);
   // var_dump($output);
   ?>
   <div class="wrap">
     <img src="<?= escape('hi there'); ?>">
   </div>
-  <?php
-}
+<?php
+}
    ----------- end diff -----------

 [OK] 1 file would have changed (dry-run) by Rector
